### PR TITLE
fix(dedicated): make swap partition removable

### PR DESCRIPTION
**Pull request by JayBeeDe**
| Question         | Answer
| ---------------- | ---
| Branch?          | `master`
| Bug fix?         | yes
| New feature?     | no
| Breaking change? | no
| Tickets          | Fix MANAGER-11056
| License          | BSD 3-Clause

- [x] Try to keep pull requests small so they can be easily reviewed.
- [x] Commits are signed-off
- [x] Only FR translations have been updated
- [x] Branch is up-to-date with target branch
- [x] Lint has passed locally <-- The linting has never been done on changed js and html files (before my change)
- [x] Standalone app was ran and tested locally
- [x] Ticket reference is mentioned in linked commits (internal only)
~~Breaking change is mentioned in relevant commits ~~

## Description

Make swap partition optional (removable) within the partitioning

## Related

ref: MANAGER-11056

### CODE REVIEW

**Review by anooparveti — DISMISSED** [review]

**Review by ghyenne — DISMISSED** [review]

**Review by sachinrameshn — APPROVED** [review]

**Comment by JayBeeDe** [comment]
@anooparveti @ghyenne @sachinrameshn sorry for the mistake, I have deleted the commit I have added by error 10 min ago

**Review by anooparveti — DISMISSED** [review]

**Comment by anooparveti** [comment]
@JayBeeDe  can you please resolve the conflicts? Thanks!

**Review by ghyenne — APPROVED** [review]

**Review by anooparveti — APPROVED** [review]

**Review by ghyenne — DISMISSED** [review]

File: packages/manager/apps/dedicated/client/app/dedicated/server/installation/ovh/dedicated-server-installation-ovh.controller.js [hunk 1]
```diff
@@ -1529,47 +1493,41 @@ angular
...
+          $scope.errorInst.mountPoint2 =
+            !$scope.errorInst.mountPointEmpty &&
+            !$scope.errorInst.mountPointUse &&
+            !$scope.errorInst.mountPoint &&
+            /^\/var\/log/.test(partition.mountPoint.toLowerCase()) &&
+            /^(ovh|gentoo-ovh_64|gentoo-ovh)$/.test(
+              $scope.installation.selectDistribution.family,
+            );
+        } else if (
+          $scope.installation.selectDistribution.family ===
```
Comment (darsene) [inline packages/manager/apps/dedicated/client/app/dedicated/server/installation/ovh/dedicated-server-installation-ovh.controller.js:1520]:
```suggestion
         else if ($scope.constants.warningWindows)
```

File: packages/manager/apps/dedicated/client/app/dedicated/server/installation/ovh/dedicated-server-installation-ovh.controller.js [hunk 2]
```diff
@@ -1529,47 +1493,41 @@ angular
...
+
+          $scope.errorInst.mountPoint2 =
+            !$scope.errorInst.mountPointEmpty &&
+            !$scope.errorInst.mountPointUse &&
+            !$scope.errorInst.mountPoint &&
+            /^\/var\/log/.test(partition.mountPoint.toLowerCase()) &&
+            /^(ovh|gentoo-ovh_64|gentoo-ovh)$/.test(
+              $scope.installation.selectDistribution.family,
+            );
+        } else if (
```
Comment (darsene) [inline packages/manager/apps/dedicated/client/app/dedicated/server/installation/ovh/dedicated-server-installation-ovh.controller.js:1519]:
```suggestion
        } else {
```

**Review by darsene — DISMISSED** [review]

**Comment by JayBeeDe** [comment]
:no_entry: issue with QA... Need to rebase and fix it.

**Comment by JayBeeDe** [comment]
Ready for re-review

**Review by ghyenne — APPROVED** [review]

**Review by darsene — APPROVED** [review]

**Pull request merged**